pkg/console/subresource/deployment/deployment.go

```diff
@@ -461,3 +464,12 @@ func customLogoVolume() volumeConfig {
 		path:        "/var/logo/",
 		isConfigMap: true}
 }
+
```

Comment:
This is to overwrite the existing serving cert with the custom, so we don't have to change the impl in the console to look at a different dir, correct?

Reply:
yes
Thought that we should not need both serving-certs